src/RecoveryServices/RecoveryServices.SiteRecovery.Test/ScenarioTests/A2A/AsrA2ATests.ps1

```diff
@@ -549,6 +549,143 @@ function Test-ReplicateProximityPlacementGroupVm{
 		Assert-NotNull($pe.providerSpecificDetails.RecoveryProximityPlacementGroupId)
 }
 
+<#
+.SYNOPSIS 
+    Test VMNicConfig new parametersets
+#>
+
+function Test-VMNicConfig{
+   param([string] $seed ='330')
+        $primaryPolicyName = getPrimaryPolicy
+        $recoveryPolicyName = getRecoveryPolicy
+        
+        $primaryContainerMappingName = getPrimaryContainerMapping
+        $recoveryContainerMappingName = getRecoveryContainerMapping
+        
+        $primaryContainerName = getPrimaryContainer
+        $recoveryContainerName = getRecoveryContainer
+        $vaultRgLocation = getVaultRgLocation
+        $vaultName = getVaultName
+        $vaultLocation = getVaultLocation
+        $vaultRg = getVaultRg
+        $primaryLocation = getPrimaryLocation
+        $recoveryLocation = getRecoveryLocation
+        $primaryFabricName = getPrimaryFabric
+        $recoveryFabricName = getRecoveryFabric
+        $RecoveryReplicaDiskAccountType = "Premium_LRS"
```

Comment:
Fix indentation. Check for similar issues everywhere else in this file.

Reply:
i can see indentation issue in PR.. but its good is VS... maybe its cause of indentation issue in other tests in file.. will fix dem and see

Reply:
done